client/src/components/Input.js

```diff
@@ -29,6 +29,6 @@ export const InputField = ({ title, value, onChange }) => {
 
 InputField.propTypes = {
   title: PropTypes.string.isRequired,
-  value: PropTypes.string.isRequired,
+  value: PropTypes.string,
```

Comment:
here you say that value is a string. if you want it to be an array you need to change that

Reply:
But I don't want that it is an array. The user type a word inside the inputfield, so in my mind it has to be a string.

I already try to change it to an array. But then the warning said: "Failed prop type: Invalid prop value of type `string` supplied to InputField, expected `array` at InputField.